### PR TITLE
add qualtrics tag code

### PR DESCRIPTION
**Pull request by leggsimon**
🐿 v2.7.0

### CODE REVIEW

File: browser/layout/partials/qualtrics-tag.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!--BEGIN QUALTRICS SITE INTERCEPT-->
```
Comment (adgad) [inline browser/layout/partials/qualtrics-tag.html]: Are there any docs/original hosting for this script that we can put in a comment?

File: browser/layout/partials/qualtrics-tag.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+				if (this.check()) {
+					var a = document.createElement("script");
+					a.type = "text/javascript";
+					a.src = g + "&t=" + (new Date()).getTime();
+					document.body && document.body.appendChild(a)
+				}
+			};
+			this.start = function () {
+				var a = this;
+				window.addEventListener ? window.addEventListener("load", function () {
```
Comment (benbarnett) [inline browser/layout/partials/qualtrics-tag.html]: why not

Comment (benbarnett) [reply browser/layout/partials/qualtrics-tag.html]: have

Comment (benbarnett) [reply browser/layout/partials/qualtrics-tag.html]: 👇

Comment (adgad) [reply browser/layout/partials/qualtrics-tag.html]: Oh good point, this should probably be checking CTM

File: browser/layout/partials/qualtrics-tag.html (hunk 1 above)
Comment (leggsimon) [reply browser/layout/partials/qualtrics-tag.html]: I'll add some later

**Pull request merged**